charter.html

```diff
@@ -233,44 +237,40 @@ <h2 id=proposals>Proposals</h2>
 <p>The Community Group may discuss the Proposal on GitHub and during
 teleconferences or face-to-face <a href=#meetings>meetings</a>.
 
-<p>The <a href=#chairs>Chairs</a> may create a dedicated repository for
-a Proposal. They may do this for any reason, such as but not limited to:
+<p>Champions are responsible for the technical content of their
+Proposal. They are expected to solicit input from Community Group
+Participants, and they may consider and respond to comments,
+suggestions, and objections on their Proposal from Participants and the
+public.
 
-<ul>
-<li>because the Proposal has become sufficiently complex that writing an
-explainer is warranted
-<li>because the Proposal is sufficiently complex that having a dedicated
-issue tracker for it is warranted
-<li>because multiple <a href=#implementer>implementers</a> have
-expressed interest in the Proposal
-</ul>
+<p>Champions may ask the <a href=#chairs>Chairs</a> to create a
+dedicated repository for their Proposal.
 
 <p>Proposals begin as or evolve into
 <a href=https://w3ctag.github.io/explainers>explainers</a> which
-describe the proposed changes to the web platform, and which may serve
-as the basis for a Specification (should the Proposal be adopted as
-one).
-
-<p class=note>While explainers are Community Group Reports as defined in
-the <a href=https://www.w3.org/community/about/agreements/>Community and
-Business Group Process</a>, they are not Specifications as defined in
-that document.
+describe a user-facing problem which needs to be solved and how the
+authors propose to solve it. Explainers are Community Group Reports as
+defined in the
+<a href=https://www.w3.org/community/about/agreements/>Community and
+Business Group Process</a>, but they are not Specifications as defined
+in that document.
 
-<p>Proposals may be withdrawn by their originators, or may be closed by
+<p>Proposals may be withdrawn by their Champions, or may be closed by
 the Chairs (if, for example, the Chairs deem the Proposal to
-be <a href=out-of-scope>out of scope</a> or the Proposal fails to gain
-sufficient <a href=#implementer>implementer</a> support to be adopted as
-a Specification). If such a Proposal has a dedicated repository, the
-Chairs should take steps to ensure the data is not lost, perhaps by
-transferring the repository to a different organization or user, or by
-archiving it.
+be <a href=out-of-scope>out of scope</a>, or if its number of Champions
+drops to zero). If such a Proposal has a dedicated repository, the
```

Comment:
This suggests someone may drop out from being a Champion. Perhaps this is implicit in "Champions are self-organizing" but maybe the Champions should at least tell the chairs when composition changes?

Reply:
I've addressed this in 2dd8162.